src/ffs/types.ts

```diff
@@ -41,7 +41,7 @@ export type Links = { [name: string]: Link }
 // ----
 
 export type NonEmptyPath = [string, ...string[]]
-export type SyncHook = (cid: CID) => any
+export type SyncHook = (cid: CID) => unknown
```

Comment:
I did not know `unknown` was a thing 🤷

Reply:
yeah just recently found this out also! my understanding is that it's like `any` but doesn't completely blow out the type system